### PR TITLE
Fix ordering of Memberlist

### PR DESCRIPTION
**Pull request by theworldbright**
Resolves https://github.com/vector-im/vector-web/issues/1492 and https://github.com/vector-im/vector-web/issues/1705

### CODE REVIEW

**Comment by theworldbright** [comment]
One second, let me adjust this.

**Comment by theworldbright** [comment]
Ok ready!

File: src/components/views/rooms/MemberList.js [hunk 1]
```diff
@@ -423,29 +423,9 @@ module.exports = React.createClass({
             if (userA.currentlyActive && !userB.currentlyActive) return -1;
             if (!userA.currentlyActive && userB.currentlyActive) return 1;
 
-            // For now, let's just order things by timestamp. It's really annoying
```
Comment (richvdh) [inline src/components/views/rooms/MemberList.js]: this comment still applies; please don't get rid of it

File: src/components/views/rooms/MemberList.js [hunk 2]
```diff
@@ -423,29 +423,9 @@ module.exports = React.createClass({
...
-            }
-    */
-
-            var lastActiveTsA = userA && userA.lastActiveTs ? userA.lastActiveTs : 0;
-            var lastActiveTsB = userB && userB.lastActiveTs ? userB.lastActiveTs : 0;
-
-            // console.log("comparing ts: " + lastActiveTsA + " and " + lastActiveTsB);
-
-            return lastActiveTsB - lastActiveTsA;
+            var lastActiveTsA = userA && userA.lastActiveAgo ? userA.lastActiveAgo : 9999999999;
```
Comment (richvdh) [inline src/components/views/rooms/MemberList.js]: `user.lastActiveAgo` is relative to `user.lastPresenceTs`. I suggest you want `user.getLastActiveTs()`.

The test on `userA` is redundant due to the tests at line 397-399.

Why 9999999999, as opposed to `Number.MAX_VALUE`, or 0?

File: src/components/views/rooms/MemberList.js (hunk 1 above)
Comment (theworldbright) [inline src/components/views/rooms/MemberList.js]: Got it and added back

**Comment by theworldbright** [comment]
Hmm currently it will simply order by timestamp regardless of whether the user is offline or not. Is this what we want?

**Comment by richvdh** [comment]
> Hmm currently it will simply order by timestamp regardless of whether the user is offline or not. Is this what we want?

Isn't that what the comment I made you reintroduce is about? (In other words: it's not ideal, but it's ok for now)

**Comment by theworldbright** [comment]
> Isn't that what the comment I made you reintroduce is about? (In other words: it's not ideal, but it's ok for now)

Yeah either way I like it better than what we currently have. Any other issues in terms of this PR?

File: src/components/views/rooms/MemberList.js (hunk 2 above)
Comment (theworldbright) [inline src/components/views/rooms/MemberList.js]: As a sidenote, thanks for teaching me about Number.MAX_VALUE. Didn't know it existed; I'm still a bit new to ECMAScript.

**Comment by richvdh** [comment]
lgtm. thanks for the PR!

**Pull request merged**